docs/getting-started/self-healing-rollback.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[self-healing-rollback]]
+= Configure self-healing rollback for Windows endpoints
+
+{endpoint-cloud-sec}'s self-healing feature rolls back file changes and processes on Windows endpoints when a prevention alert is generated by enabled protection features. All activity on the host reverts to its state five minutes before the prevention alert.
+
+This can help contain the impact of malicious activity, as {endpoint-cloud-sec} not only stops the activity but also erases any attack artifacts deployed prior to detection.
+
+Self-healing rollback is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature, and it's only supported for Windows endpoints.
+
+[CAUTION]
+====
+This feature can cause data loss, since it reverts _all_ recent changes on the host, not just changes directly related to a threat.
```

Comment:
In its current implementation yes permanent data loss can occur because we delete recently added files or overwrite changes with an old copy. 

It is not accurate here that _all_ recent changes are reverted. For example, we have a set of filters which limit targeted files to paths which are most likely related to the threat. However, it is accurate to say that there can be incidental actions that are unrelated to the threat.

Reply:
@joe-desimone @nastasha-solomon Let me know if this conveys those nuances:
```suggestion
This feature can cause permanent data loss, since it overwrites recent changes and deletes recently added files on the host. Self-healing rollback targets the changes related to a detected threat, but it may also include incidental actions that aren't directly related to the threat.
```